shared/src/data/default-state/patient-templates.ts

```diff
@@ -12,173 +13,1210 @@ const defaultMaleImage: ImageProperties = {
+        },
+    ]
+);
+
+const greenUntilPhase8State = PatientHealthState.create(
+    FunctionParameters.create(
+        calculateHealthChange(
+            healthPointsDefaults.greenMax,
+            healthPointsDefaults.yellowAverage,
+            8
+        ),
+        0,
+        0,
+        0
+    ),
+    [
+        {
+            matchingHealthStateId: waitForYellowToGreenState.id,
+            maximumHealth: healthPointsDefaults.yellowAverage,
+        },
+    ]
+);
+
+const greenStartPhase11RADecisionState = PatientHealthState.create(
+    FunctionParameters.create(
+        calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.blackMax,
+            1
+        ),
+        calculateHealthChange(
+            healthPointsDefaults.redMax,
+            healthPointsDefaults.greenMax,
+            1
+        ),
+        calculateHealthChange(
+            healthPointsDefaults.redMax,
+            healthPointsDefaults.greenMax,
+            1
+        ),
+        0
+    ),
+    [
+        {
+            matchingHealthStateId: noChangesState.id,
+            minimumHealth: healthPointsDefaults.greenAverage,
+        },
+    ]
+);
+
+const greenStartPhase10RADecisionState = PatientHealthState.create(
+    FunctionParameters.create(
+        calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        ),
+        calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.greenMax,
+            1
+        ),
+        calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.greenMax,
+            1
+        ),
+        0
+    ),
+    [
+        {
+            matchingHealthStateId: noChangesState.id,
+            minimumHealth: healthPointsDefaults.greenMax,
+        },
+        {
+            matchingHealthStateId: greenStartPhase11RADecisionState.id,
+            earliestTime: phaseTime,
+        },
+    ]
+);
+
+const greenStartPhase9RSDecisionState = PatientHealthState.create(
+    FunctionParameters.create(
+        calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        ),
+        calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.greenAverage,
+            1
+        ) -
+            calculateHealthChange(
+                healthPointsDefaults.yellowAverage,
+                healthPointsDefaults.redAverage,
+                1
+            ),
+        -calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        )
```

Comment:
Why are you using negations here?

Reply:
You are actually using this more below, but not above.

Reply:
In these cases personnel just holds the current healthpointlevel, in the other cases they actually can increase them